Makefile

```diff
@@ -20,22 +20,34 @@ $(error Environment variable BOLOS_SDK is not set)
 endif
 include $(BOLOS_SDK)/Makefile.defines
 
-APP_LOAD_PARAMS= --curve ed25519 --path "44'/508'" --appFlags 0x240 $(COMMON_LOAD_PARAMS)
+APP_LOAD_PARAMS = --curve ed25519
+ifeq ($(TARGET_NAME), TARGET_NANOX)
+APP_LOAD_PARAMS += --appFlags 0x240  # APPLICATION_FLAG_BOLOS_SETTINGS
+else ifeq ($(TARGET_NAME), TARGET_STAX)
+APP_LOAD_PARAMS += --appFlags 0x240  # APPLICATION_FLAG_BOLOS_SETTINGS
```

Comment:
Indeed it could be factorized but I prefer keeping it as flat as possible and as close to the boilerplate example.
We plan to abstract all this from the app eventually anyway

Reply:
Done